modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -153,6 +155,75 @@ public function filter_salt( $salt, $scheme ) {
 
 		return $salt;
 	}
+
+	/**
+	 * Enqueue the scripts for the WordPress.com block editor integration.
+	 */
+	public function enqueue_scripts() {
+		$debug         = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG;
+		$version       = gmdate( 'YW' );
+		$is_calypsoify = 1 === (int) get_user_meta( get_current_user_id(), 'calypsoify', true );
+
+		$src_common = $debug
+			? '//widgets.wp.com/wpcom-block-editor/common.js?minify=false'
+			: '//widgets.wp.com/wpcom-block-editor/common.min.js';
+		wp_enqueue_script(
```

Comment:
Should this be enqueued when Calypsoify is not active?

Reply:
At least is what we do in WordPress.com but I'm not sure we have to do it here. The `wpcom-block-editor` scripts provide integrations for two environments: Calypso iframed block editor and WP Admin block editor ([#](https://github.com/Automattic/wp-calypso/tree/master/apps/wpcom-block-editor#environments)).

The `wpcom-block-editor/calypso-*.js` scripts enhances the Calypso iframed block editor replacing core tools like media modal or revisions which Calypso tools, which is why we only load when Calypsoify is active (my last commit changes this check in favor of the new `is_iframed_block_editor` method to make clear is not Calypsoify but the iframed block editor).

The `wpcom-block-editor/common.js` scripts enhances both editor, which is why we load them always. These enhancements add some missing features in Core like justify or underline text formats, or the ability to switch to the classic editor from the block editor. I think it makes sense to have these features on a Jetpack site connected to WordPress.com, so users will have the same experience in both envs.